tests/starwars/test_dsl.py

```diff
@@ -84,67 +98,6 @@ def test_fetch_luke_query(ds):
     assert query == str(query_dsl)
 
 
-# def test_fetch_some_id_query():
```

Comment:
I guess the idea was to implement these tests with gql. So I would leave them there as a reminder.

Reply:
Those tests already exist on the `test_query.py` file, which make use of gql instead of plain string, that's why I removed them.

Reply:
Ah, ok then.